libs/interpret-vision/src/lib/VisionExplanationDashboard/Controls/Flyout.tsx

```diff
@@ -25,30 +25,65 @@ export interface IFlyoutProps {
   isOpen: boolean;
   item: IVisionListItem | undefined;
   loadingExplanation: boolean;
+  otherMetadataFieldName: string;
```

Comment:
I think this should be a list of fieldnames instead of just one here?

Reply:
will be overwritten by model assessment PR